daisy_workflows/image_build/centos/centos6.wf.json

```diff
@@ -4,20 +4,17 @@
     "image-dest": "",
     "installer-iso": ""
   },
-  "Sources": {
-    "installer.iso": "${installer-iso}"
-  },
   "Steps": {
     "build-centos6": {
       "Timeout": "60m",
       "SubWorkflow": {
         "Path": "./centos.wf.json",
         "Vars": {
           "image-dest": "${image-dest}",
-          "installer-iso": "${SOURCESPATH}/installer.iso",
+          "installer-iso": "${installer-iso}",
           "license-id": "1000206",
           "release": "centos6",
-          "wf-name": "build-centos6"
+          "wf-name": "centos6"
```

Comment:
What is this used for? I think just using a standard subworkflow name (instead of a var) would work fine.